README.md

```diff
@@ -13,160 +16,3 @@ Tools for working with LAMMPS data and dump files.
 or `conda`:
 
     conda install -c conda-forge lammpsio
-
-## Snapshot
-
-The particle configuration is stored in a `Snapshot`. A `Snapshot` holds the
-data for *N* particles, the simulation `Box`, and the timestep. The `Box` follows
-the LAMMPS conventions for its shape and bounds. Here is a 3-particle
-configuration in an orthorhombic box centered at the origin at step 100:
-
-    box = lammpsio.Box((-2,-3,-4), (2,3,4))
-    snapshot = lammpsio.Snapshot(3, box, step=100)
-
-These constructor arguments are available as attributes:
-
-- `N`: number of particles (int)
-- `box`: bounding box (`Box`)
-- `step`: timestep counter (int)
-- `num_types`: number of particle types (int). If `num_types is None`, then the number of types is deduced from `typeid`.
-
-The data contained in a `Snapshot` per particle is:
-
-- `id`: (*N*,) array atom IDs (dtype: `int`, default: runs from 1 to *N*)
-- `position`: (*N*,3) array of coordinates (dtype: `float`, default: `(0,0,0)`)
-- `image`: (*N*,3) array of periodic image indexes (dtype: `int`, default: `(0,0,0)`)
-- `velocity`: (*N*,3) array of velocities (dtype: `float`, default: `(0,0,0)`)
-- `molecule`: (*N*,) array of molecule indexes (dtype: `int`, default: `0`)
-- `typeid`: (*N*,) array of type indexes (dtype: `int`, default: `1`)
-- `mass`: (*N*,) array of masses (dtype: `float`, default: `1`)
-- `charge`: (*N*,) array of charges (dtype: `float`, default: `0`)
```

Comment:
All these default values need to be documented in the `Snapshot`. See what I did to `Topology` for an example.